terraform_compliance/steps/then/its_value_condition_be_null.py

```diff
@@ -0,0 +1,39 @@
+from terraform_compliance.common.error_handling import Error
+from terraform_compliance.common.helper import Null
+import re
+
+
+def its_value_condition_be_null(_step_obj, condition):
+    if condition not in ['must', 'must not']:
+        Error(_step_obj, 'Condition must be one of "must" or "must not"')
+        return
+
+    condition_must = condition == 'must'
+
+    match = _step_obj.context.match
+    regex = r'{}'.format(u'(\x00|^$)$')
+    is_null = lambda x: x in ([], {}, None, Null) or (isinstance(x, str) and match.regex_match(regex, x))
+
+    for resource in _step_obj.context.stash:
+        values = resource.get('values', {})
```

Comment:
Smells legacy code. Is it even possible to not have values in a resource anymore?